src/index.html

```diff
@@ -11,7 +11,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class =" form-field">
+        <legend>Personal information</legend>
+
+        <label class="input-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="input-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+        </label>
+
+        <label class="input-field">
+          How old are You?
+          <input
+            type="number"
+            name="old"
+            value="12"
+            max="100"
+            min="10"
+            >
+        </label>
+
+        <label class="input-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateBirth"
+            >
+        </label>
+
+        <label class="input-field terms">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+            >
```

Comment:
```suggestion
          >
```
[CHECKLIST](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
Rule - 3

If you have a problem with autoformatting, you can find solutions here:
[mate-academy.github.io/layout_task-guideline/#send-your-code-to-github-by-running-codegit-push-origin-developcode](https://mate-academy.github.io/layout_task-guideline/#send-your-code-to-github-by-running-codegit-push-origin-developcode)

Reply:
Changes have been added